pkg/ebpf/probes/probes.go

```diff
@@ -138,4 +139,6 @@ const (
 	SignalSchedProcessFork
 	SignalSchedProcessExec
 	SignalSchedProcessExit
+	ExecuteFinisehd
+	ExecuteAtFinisehd
```

Comment:
ditto and for all occurrences below...

```suggestion
	ExecuteFinished
	ExecuteAtFinished
```